From 707820f027d91588dd8dac9299580809d93329d5 Mon Sep 17 00:00:00 2001
From: "yi.zhang1" <yi.zhang1@amlogic.com>
Date: Thu, 13 Apr 2023 17:19:52 +0800
Subject: [PATCH] amlbian: disable some useless service  [1/1]

PD#SWPL-119086

Problem:
amlbian some service start failed.

Solution:
disable some useless service.

Verify:
local

Signed-off-by: yi.zhang1 <yi.zhang1@amlogic.com>
Change-Id: I0213f34d845cbf04a583d5fe792b8ef855e2377b
---
 config/Debian/distribution-special-function | 4 +++-
 config/functions/build-rootfs               | 3 +++
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/config/Debian/distribution-special-function b/config/Debian/distribution-special-function
index 3e160d3..cfbbd35 100755
--- a/config/Debian/distribution-special-function
+++ b/config/Debian/distribution-special-function
@@ -102,7 +102,9 @@ install_distribution_specific()
 	esac
 }
 
-
+post_install_distribution_specific()
+{
+}
 
 
 ## tweaks for distribution
diff --git a/config/functions/build-rootfs b/config/functions/build-rootfs
index 61d3588..e78f9b7 100755
--- a/config/functions/build-rootfs
+++ b/config/functions/build-rootfs
@@ -151,6 +151,9 @@ install_common()
 	build_gnu_deb
 	install_gnu_deb
 
+    #disable some useless service
+    post_install_distribution_specific
+
 	# post install
 	shell_chroot "echo PATH=${PATH}"
 
-- 
2.25.1

